core/request.go

```diff
@@ -104,11 +106,12 @@ func (r *RequestAccessor) StripBasePath(basePath string) string {
 
 // ProxyEventToHTTPRequest converts an API Gateway proxy event into an
 // http.Request object.
-// Returns the populated request with an additional two custom headers for the
-// stage variables and API Gateway context. To access these properties use
-// the GetAPIGatewayStageVars and GetAPIGatewayContext method of the RequestAccessor
-// object.
-func (r *RequestAccessor) ProxyEventToHTTPRequest(req events.APIGatewayProxyRequest) (*http.Request, error) {
+// Returns the populated request with:
+// 		* an additional two custom headers for the stage variables and API Gateway context.
```

Comment:
Whitespace mis-alignment. You might make sure there are no internal `\t` bytes.

Reply:
Corrected